src/promptflow-devkit/promptflow/_cli/_pf/_flow.py

```diff
@@ -519,18 +518,19 @@ def _test_flow_multi_modal(args, pf_client):
         from promptflow._sdk._tracing import _invoke_pf_svc
 
         pfs_port = _invoke_pf_svc()
+        serve_app_port = find_available_port()
         flow = generate_yaml_entry_without_delete(entry=args.flow)
         # flex flow without yaml file doesn't support /eval in chat window
-        enable_internal_features = Configuration.get_instance().is_internal_features_enabled() or flow != args.flow
-        chat_page_url = construct_chat_page_url(
-            flow,
-            pfs_port,
-            list_of_dict_to_dict(args.url_params),
+        enable_internal_features = Configuration.get_instance().is_internal_features_enabled() and flow == args.flow
```

Comment:
why change this?
flow != args.flow
->
flow == args.flow

Reply:
this is a request from @YingChen1996: [link](https://github.com/microsoft/promptflow/pull/2927#discussion_r1596536561)

This is to control whether to show experiment tab and it seems that previous logic is wrong.